manifests/init.pp

```diff
@@ -1,154 +1,117 @@
-# @summary Installs and configures Grafana with InfluxDB for monitoring Puppet infrastructure.
```

Comment:
This is mostly grouping sorting, with some corrections to defaults that drifted over time between the comments and `data/common.yaml`.